static/sass/_snapcraft_p-build.scss

```diff
@@ -21,6 +23,17 @@
     height: 30px;
     vertical-align: middle;
   }
-}
 
+  .p-icon--restart {
+    @extend %icon;
+    background-image: url("#{$assets-path}34f54cf0-restart.svg");
```

Comment:
We should probably add a little comment that says to switch to the vanilla icon when these get added to vanilla.